Ktisis/Util/MathHelpers.cs

```diff
@@ -92,5 +94,11 @@ public class MathHelpers {
 
 			return angle;
 		}
+
+		public static UInt32 ConvertVector4ToUInt32(Vector4 color)
```

Comment:
Is this method actually necessary? It looks like it can be trivially inlined at no real cost to readability.